external/mcpclientplugin/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@microsoft/spark.mcpclientplugin",
```

Comment:
none of the plugin packages have plugin in the name, I suggest `@microsoft/spark.mcpclient` or `@microsoft/spark.mcp.client`, although I'd still like to see if we can merge the other mcp package with this one someone and use named exports so people can just do `import ... from '@microsoft/spark.mcp/{server | client}'`

Reply:
Will update to use mcpClient, but let's keep it separate for simplicity's sake for now, unless you feel strongly